app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h3>Invoice <%=@invoice.id %></h3>
+
+<div id="invoice-details">
```

Comment:
I like this, I should implement it in the equivalents on my end to do some within testing

Reply:
Can do some nice nested within blocks as well, if you dare haha